bokeh/application/handlers/tests/test_code.py

```diff
@@ -58,22 +57,6 @@ def test_script_runtime_error(self):
         assert handler.error is not None
         assert 'nope' in handler.error
 
-    def test_script_sys_path(self):
-        doc = Document()
-        handler = CodeHandler(source="""import sys; raise RuntimeError("path: '%s'" % sys.path[0])""", filename="/test_filename")
-        handler.modify_document(doc)
-
-        assert handler.error is not None
-        assert "path: ''" in handler.error
-
-    def test_script_cwd(self):
-        doc = Document()
-        handler = CodeHandler(source="""import os; raise RuntimeError("cwd: '%s'" % os.getcwd())""", filename="/test_filename")
-        handler.modify_document(doc)
-
-        assert handler.error is not None
-        assert "cwd: '%s'" % abspath(sep) in handler.error
-
```

Comment:
Can you elaborate? These (or something like them) seem very important to maintain.